### PR TITLE
I tried to sort out merge conflicts for https://github.com/OpenMined/PySyft/pull/5935

### PR DESCRIPTION
**Pull request by iamtrask**

### CODE REVIEW

**Comment by iamtrask** [comment]
Not sure what's broken at present - current error when launching with hagrid:

```
 => [openmined/grid-backend:0.6.0-alpha.0 backend  8/10] COPY --from=build /root/.local /root/.local                                                             22.5s
 => ERROR [openmined/grid-backend:0.6.0-alpha.0 backend  9/10] COPY --from=build /usr/local/bin/waitforit /usr/local/bin/waitforit                                0.2s
------
 > [openmined/grid-backend:0.6.0-alpha.0 backend  9/10] COPY --from=build /usr/local/bin/waitforit /usr/local/bin/waitforit:
------
failed to solve: rpc error: code = Unknown desc = failed to add snapshot lilmhs3ftrssddu76ed3deggm to lease: input/output error
```

**Comment by iamtrask** [comment]
False alarm - I just needed to clear out my docker volumes and rebuild.

File: packages/grid/backend/grid/api/router.py [hunk 1]
```diff
@@ -2,23 +2,20 @@
...
+from grid.api.setup import setup
+from grid.api.syft import syft
+from grid.api.users.routes import router as user_router
 
 api_router = APIRouter()
 api_router.include_router(login.router, tags=["login"])
 api_router.include_router(user_router, prefix="/users", tags=["users"])
 # comment out in merge
-api_router.include_router(requests_router, prefix="/requests", tags=["requests"])
+# api_router.include_router(requests_router, prefix="/requests", tags=["requests"])
```
Comment (madhavajay) [inline packages/grid/backend/grid/api/router.py:18]: Why is this commented out? Is it not being used?

File: packages/grid/frontend/frontend.dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+COPY . .
+
+FROM node:16-alpine as grid-ui-development
+
+ENV NEXT_TELEMETRY_DISABLED=1
+
+WORKDIR /app
+COPY --from=init-stage /app .
+CMD ["/usr/local/bin/node", "--max_old_space_size=4096", "/app/node_modules/.bin/next", "dev", "-p", "80"]
```
Comment (madhavajay) [inline packages/grid/frontend/frontend.dockerfile:18]: Seems like this is now kebab case: https://nodejs.org/api/cli.html#cli_max_old_space_size_size_in_megabytes

**Review by madhavajay — APPROVED** [review]
Excellent work. This is good to go when the tests pass.

**Pull request merged**